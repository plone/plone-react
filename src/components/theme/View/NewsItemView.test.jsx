import React from 'react';
import renderer from 'react-test-renderer';
import NewsItemView from './NewsItemView';

test('renders a news item view component', () => {
  const component = renderer.create(
    <NewsItemView
      content={{
        title: 'Hello World!',
        description: 'Hi',
        text: {
          data: '<p>Hello World!',
        },
      }}
    />,
  );
  const json = component.toJSON();
  expect(json).toMatchSnapshot();
});
