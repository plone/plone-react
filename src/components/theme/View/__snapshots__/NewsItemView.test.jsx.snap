// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a news item view component 1`] = `
<div
  className="ui container view-wrapper"
>
  <h1
    className="documentFirstHeading"
  >
    Hello World!
  </h1>
  <p
    className="documentDescription"
  >
    Hi
  </p>
  <div
    dangerouslySetInnerHTML={
      Object {
        "__html": "<p>Hello World!",
      }
    }
  />
</div>
`;
